src/stories/styleguide.mdx

```diff
@@ -0,0 +1,70 @@
+# Welcome to the GreenwoodJS style guide
+
+### The GreenwoodJS website styleguide breaks down into the following sections:
+
+1. Colors
+2. Typography
+3. Spacing
+4. Components
+
+
+
+## Colors
+The following theme colors available are:
```

Comment:
We'll definitely want to flesh this section out with examples like in the reference storybook
https://raw.githubusercontent.com/AnalogStudiosRI/www.blissfestri.com/main/src/stories/Styleguide.stories.mdx